src/Quartz/Impl/Calendar/DailyCalendar.cs

```diff
@@ -105,6 +105,8 @@ public class DailyCalendar : BaseCalendar
         ///     </li>  
         /// </ul>
         /// </summary>
+        /// <param name="rangeStartingTime">The range starting time in millis.</param>
```

Comment:
i'm pretty sure this is the normal way to document parameters. `<param>` is used to describe parameters and `<paramref>` is used inside `<summary>` blocks to reference parameters by name.